docs/source/theory/fktables.rst

```diff
@@ -0,0 +1,130 @@
+.. _fktables:
+
+============================================================
+Fast Interface (FK tables)
+============================================================
+
+.. raw:: latex
+
+   \maketitle
+
+.. raw:: latex
+
+   \tableofcontents
+
+Here we discuss the numerical implementation of the calculations of both the DIS structure functions
+and the hadronic cross-sections.
+
+In the framework of collinear QCD factorization, the :math:`F_2` structure function
+can be decomposed in terms of hard-scattering coefficient functions and PDFs as,
+
+.. math::
+
+    \begin{align} 
+    \label{eq:ev} 
+    F_2(x,Q^2) &= \sum_i^{n_f} C_i(x,Q^2) \otimes f_i(x,Q^2) \nonumber \\
```

Comment:
- let's go full eko notation: so represent flavor space by bold
- I'd also rather suppress the x-es on the rhs - because they are not the x from the lhs

Reply:
I agree but I believe that the meaning of this notation is that after the convolution you are left with the dependence on x. So what I can do is to write something like (C(Q) \otimes f(Q) )(x). What do you think?

Reply:
and regarding the flavor space, do you mean the n_f? or the PDFs (and the ekos) in flavor basis?

Reply:
- flavor: suppress all flavor stuff: sum, i, nf
- x: mmm, I'd rather have no x-es at all, if needs be suppress also on the left; what do you think @andreab1997  @AleCandido ?

Reply:
ok, for the moment I remove the x